neurom/apps/morph_stats.py

```diff
@@ -181,13 +181,17 @@ def _fill_stats_dict(data, stat_name, stat, shape):
         neurite_type = _NEURITE_MAP[neurite_type]
         feature, func = _get_feature_value_and_func(feature_name, neurons,
                                                     neurite_type=neurite_type)
+        if np.isscalar(feature):
```

Comment:
with this you will allow : builtin string and buffer objects. Do you need this ? 
Otherwise I would recomend using : 
```
isinstance(feature, numbers.Number)
```

Reply:
That's a good question. If we have string or any other non-number object then there will be error anyway, as  only numbers or list of numbers are expected here. I will update to your suggestion.